pkg/cloud/services/compute/instance.go

```diff
@@ -608,8 +613,9 @@ func (s *Service) DeleteInstance(eventObject runtime.Object, instanceName string
 		return nil
 	}
 
+	mc := metrics.NewMetricPrometheusContext("attach_interface", "list")
```

Comment:
This is on the [`/servers/server_id/os-interface` endpoint](https://docs.openstack.org/api-ref/compute/?expanded=list-port-interfaces-detail#list-port-interfaces). Xref [kubernetes/cloud-provider-openstack](https://github.com/kubernetes/cloud-provider-openstack/blob/ab36705fb740299c57cfeeefec6f4185e5f68579/pkg/openstack/openstack.go#L573)

```suggestion
	mc := metrics.NewMetricPrometheusContext("server_os_interface", "list")
```